NOTICE

```diff
@@ -45,7 +45,7 @@ modified copy of the PCollections library by Harold Cooper.
   * Homepage: https://github.com/hrldcpr/pcollections
 
 This product contains a modified portion of the 'org.springframework.lang'
-package in the Spring Framework library, distributed by Pivotal, Inc:
+package in the Spring Framework library, distributed by VMware, Inc:
```

Comment:
This was intentional to match Spring Framework's [notice.txt](https://github.com/spring-projects/spring-framework/blob/8c9c59eba0bf1d8efd01d4130e46f158c5bf9df0/src/docs/dist/notice.txt#L2).
Either is probably fine, but I'm not a lawyer.